Application/EdFi.Ods.Api/Startup/ApiBehaviorOptionsConfigurator.cs

```diff
@@ -34,7 +32,7 @@ public void Configure(ApiBehaviorOptions options)
     {
         options.InvalidModelStateResponseFactory = actionContext =>
         {
-            if (actionContext.ModelState.ValidationState == ModelValidationState.Invalid)
+            if (actionContext.ModelState.ErrorCount > 0)
```

Comment:
The behavior started after [PR 903](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/903), which I _think_ is now modifying the `UrlQueryParametersRequest` after it's been validated. This changes the overall model state to `Unvalidated` (instead of `Invalid` if there was a validation error).
So, instead of checking the `ValidationState`, I now check if there are validation errors.